### ISSUE TITLE
Redirect cn.eslint.org to zh-hans.eslint.org site

### ISSUE THREAD

**Issue by kecrily** [eslint/zh-hans.docs.eslint.org#81]
The new Simplified Chinese site is ready, let's say goodbye to cn.eslint.org. Now it's time to take people to the new site.

**Comment by nzakas** [comment]
It looks like there's no easy way to do this within Netlify DNS, so investigating other alternatives.

**Comment by kecrily** [comment]
It might be possible to configure this using the `_redirects` file

https://docs.netlify.com/routing/redirects/

**Comment by nzakas** [comment]
Yes, that's what I'm doing. I'm going to point both docs.eslint.org and cn.eslint.org to eslint.org and then use redirects to get them to the right place.
